storage_broker/src/bin/storage_broker.rs

```diff
@@ -369,7 +369,9 @@ impl BrokerService for Broker {
                     Err(RecvError::Lagged(skipped_msg)) => {
                         missed_msgs += skipped_msg;
                         if let Poll::Ready(_) = futures::poll!(Box::pin(warn_interval.tick())) {
-                            warn!("dropped {} messages, channel is full", missed_msgs);
+                            warn!("subscription id={}, key={:?}, addr={:?} dropped {} messages, channel is full",
+                            subscriber.id, subscriber.key, subscriber.remote_addr, missed_msgs);
+                            Err(Status::new(Code::Internal, "full channel"))?;
                             missed_msgs = 0;
```

Comment:
Looks a bit odd, let's improve it a bit, before merging?

* `missed_msgs = 0;` after the "return" code looks odd
* `Err(..)?;` looks like a super implicit return, let's write `return Err();` instead
* `subscriber.id, subscriber. ...` like could use an extra indent

Reply:
Plus one for 1 and 3

Reply:
It's a hotfix, not intended to stay long.

2 doesn't work in try_stream.